### PR TITLE
Added missing unit tests for deprecated aliases and missing visual tests. - RSC-778

### PR DESCRIPTION
**Pull request by entroform**
https://issues.sonatype.org/browse/RSC-778

### CODE REVIEW

**Comment by rpokorny** [comment]
I don't see the unit tests

**Comment by entroform** [comment]
> I don't see the unit tests

I thought i had them: updated here: a80598d351de2be2aaaae4b2575676b09c6ff618

File: lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx [hunk 1]
```diff
@@ -20,6 +22,10 @@ describe('NxCollapsibleItems', function() {
       },
       getShallowComponent = enzymeUtils.getShallowComponent(NxCollapsibleItems, minimalProps);
 
+  it('is aliased as NxTreeView', function() {
+    expect(NxCollapsibleItems).toEqual(NxTreeView);
```
Comment (rpokorny) [inline lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx:26]: We want to be sure that we're checking reference equality here

```suggestion
    expect(NxCollapsibleItems).toBe(NxTreeView);
```

Comment (rpokorny) [reply lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx]: > We want to be sure that we're checking reference equality here

My intent was that you make that change on all of them

Comment (entroform) [reply lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx]: Ah yes, updated: 4406b5aa79e338ecc25d69d7839077ccc59accf8

**Review by rpokorny — APPROVED** [review]

**Pull request merged**